js/forum/src/addSubscriptionSettings.js

```diff
@@ -13,5 +13,13 @@ export default function() {
         onchange: this.preferenceSaver('followAfterReply')
       })
     );
+
+    items.add('followNewDiscussions',
+      Switch.component({
+        children: 'Follow new discussions automatically',
```

Comment:
Done: https://github.com/flarum/flarum-ext-english/pull/75

BTW: I don't think it's a good idea to save the translations of all plugins in the same repository. Each plugin should manage its own translations.